README.md

```diff
@@ -30,16 +30,58 @@ The `PYTHONASYNCIODEBUG` is important to check we have no pending asyncio tasks
 
 Please note that running tests empties the database. DO NOT RUN TESTS in production.
 
-# Generating an OpenAPI document
+# Docker Setup (Recommended)
+
+The easiest way to get started with Folksonomy API is to use Docker. This approach requires minimal setup and provides a consistent development environment.
```

Comment:
I'm not sure it's the easiest way. It really depends on who use it.

```suggestion
An easy way to get started with Folksonomy API is to use Docker (if you don't mind using it). This approach requires minimal setup and provides a consistent development environment.
```

Reply:
Indeed I think you should create a local_settings_docker.py which only rely on the environment (which is the right way to do it). I'm not sure we even need this, because I think we already made most settings use environment.